docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: aria
```

```suggestion
    environment:
      - BOT_TOKEN
      - PREFIX
      - POSTGRES_USER=postgres
      - POSTGRES_PASSWORD=postgres
      - POSTGRES_DB=aria
```
こうしないと docker の外から BOT_TOKEN と PREFIX が入ってこないはずです